recipes/acryl-datahub/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "acryl-datahub" %}
+{% set version = "0.15.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/acryl_datahub-{{ version }}.tar.gz
+  sha256: 6b1b8591bb485b2aa51010052758e9a7b2dee2030c8b373b72e944c570935f28
+
+build:
+  entry_points:
+    - datahub = datahub.entrypoints:main
+  skip: True  # [win or osx or py>=312 or py<38]
```

Comment:
It's not clear to me why this won't run on osx given the metadata in the upstream `setup.py`. In general though, I think this can be `noarch: python` and this `skip` section can be removed.

Reply:
Removed win, osx and just keeping python skips